scripts/README.md

```diff
@@ -4,3 +4,26 @@
 
 To import all the application jsons from the application store for a specific `round_id` you run (from project root):
 `python -m scripts.import_from_application --roundid={your round id}`
+
+## Populate Location Data
+
+This script does the following:
+
+1. Extracts postcode from the 'address of asset' field in the application form
+1. Sends these postcodes to postcodes.io to retrieve further location information, eg. country
+1. Updates the database, populating `assessment_records.location_json_blob` with a json blob containing the location details
+1. Writes the retrieved location data out to a csv file at the specified location.
```

Comment:
think this should be incrementing. Lovely docs though 👍

Reply:
That happens anyway - you just use `1.` every time and it increments them when it renders. Saves having to renumber everything if you add a new bullet mid-list :)

Reply:
ah very fancy, ill have to remember that one 👌